ir/src/ir.rs

```diff
@@ -0,0 +1,37 @@
+//! Serde types for font IR. See TODO:PublicLink.
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
+pub struct Axis<'a> {
+    tag: &'a str,
```

Comment:
I know this is just scaffolding but this should likely be an owned String.

Reply:
Done